### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by q1vs5k5** [lippman1125/mtcnn_caffe#1]
Hello, I'm sorry to bother you and thank for your sharing.
when I train mtcnn with your code, the output log information is as follows：

I0127 15:10:23.026201  7868 solver.cpp:258]     Train net output #0: cls_Acc = 0.25
I0127 15:10:23.026216  7868 solver.cpp:258]     Train net output #1: cls_loss = 0.693148 (* 1 = 0.693148 loss)
I0127 15:10:23.026226  7868 solver.cpp:258]     Train net output #2: pts_loss = 0.0566341 (* 0.5 = 0.0283171 loss)
I0127 15:10:23.026233  7868 solver.cpp:258]     Train net output #3: roi_loss = 0.0424632 (* 0.5 = 0.0212316 loss)
I0127 15:10:23.026242  7868 sgd_solver.cpp:112] Iteration 2211000, lr = 1e-06
I0127 15:11:10.744346  7868 solver.cpp:239] Iteration 2212000 (20.9604 iter/s, 47.709s/1000 iters), loss = 0.743386
I0127 15:11:10.744577  7868 solver.cpp:258]     Train net output #0: cls_Acc = 0.75
I0127 15:11:10.744592  7868 solver.cpp:258]     Train net output #1: cls_loss = 0.693147 (* 1 = 0.693147 loss)
I0127 15:11:10.744601  7868 solver.cpp:258]     Train net output #2: pts_loss = 0.0577579 (* 0.5 = 0.0288789 loss)
I0127 15:11:10.744609  7868 solver.cpp:258]     Train net output #3: roi_loss = 0.0427659 (* 0.5 = 0.021383 loss)

the output information is always between the above， It looks like there are some problems in this process, but I can't find for a while, can you or anybody here give me some advice by your experience?

**Comment by 17759205390** [comment]
> Hello, I'm sorry to bother you and thank for your sharing.
> when I train mtcnn with your code, the output log information is as follows：
> 
> I0127 15:10:23.026201 7868 solver.cpp:258] Train net output #0: cls_Acc = 0.25
> I0127 15:10:23.026216 7868 solver.cpp:258] Train net output #1: cls_loss = 0.693148 (* 1 = 0.693148 loss)
> I0127 15:10:23.026226 7868 solver.cpp:258] Train net output #2: pts_loss = 0.0566341 (* 0.5 = 0.0283171 loss)
> I0127 15:10:23.026233 7868 solver.cpp:258] Train net output #3: roi_loss = 0.0424632 (* 0.5 = 0.0212316 loss)
> I0127 15:10:23.026242 7868 sgd_solver.cpp:112] Iteration 2211000, lr = 1e-06
> I0127 15:11:10.744346 7868 solver.cpp:239] Iteration 2212000 (20.9604 iter/s, 47.709s/1000 iters), loss = 0.743386
> I0127 15:11:10.744577 7868 solver.cpp:258] Train net output #0: cls_Acc = 0.75
> I0127 15:11:10.744592 7868 solver.cpp:258] Train net output #1: cls_loss = 0.693147 (* 1 = 0.693147 loss)
> I0127 15:11:10.744601 7868 solver.cpp:258] Train net output #2: pts_loss = 0.0577579 (* 0.5 = 0.0288789 loss)
> I0127 15:11:10.744609 7868 solver.cpp:258] Train net output #3: roi_loss = 0.0427659 (* 0.5 = 0.021383 loss)
> 
> the output information is always between the above， It looks like there are some problems in this process, but I can't find for a while, can you or anybody here give me some advice by your experience?

@Royzon my weixin is 17759205390
